nekoyume/Assets/_Scripts/BlockChain/Agent.cs

```diff
@@ -418,10 +418,6 @@ private void InitAgent(Action<bool> callback, PrivateKey privateKey, CommandLine
                 }
 
                 // 상점의 상태를 한 번 동기화 한다.
-                States.Instance.SetShopState(
```

Comment:
Is it okay that we remove it?

Reply:
> Is it okay that we remove it?

It calls `SetShopState()` every time you enter the shop, so it doesn't seem necessary here. :thinking_face: